transport/http2_server.go

```diff
@@ -432,6 +436,20 @@ func (t *http2Server) getStream(f http2.Frame) (*Stream, bool) {
 	return s, true
 }
 
+// adjustWindow sends out extra window update over the initial window size
+// of stream if the application is requesting data larger in size than
+// the window.
+func (t *http2Server) adjustWindow(s *Stream, n uint32) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if s.state == streamDone {
+		return
+	}
+	if w := s.fc.maybeAdjust(n); n > 0 {
```

Comment:
Same as above re: `w` and `n`.

Can this function be shared somehow?  Should it be a method on the Stream instead of server/client?